portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -195,8 +195,9 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 			<liferay-ui:message arguments="<%= new Object[] {numberFormat.format(start + 1), numberFormat.format(end), numberFormat.format(total)} %>" key="showing-x-to-x-of-x-entries" />
 		</p>
 
+		<nav aria-label="pagination">
 		<ul class="pagination">
-			<li class="page-item <%= (cur > 1) ? StringPool.BLANK : "disabled" %>">
+			<span class="page-item <%= (cur > 1) ? StringPool.BLANK : "disabled" %>">
```

Comment:
`<ul>` children must be `<li>` elements.  This will ensure that the mark up content is semantic. Valid semantics create content that is reusable and more meaningful to assistive technologies.

Reply:
I attempted to use aria-hidden="true" in the <li> elements, but the screen reader continues to include these elements in the count.

Reply:
I would say that it is fine for the screen reader to include the element, however it won't be interactive (no <a> element inside).

Reply:
From the description of the task -> "If we need to hide next/last page link, just change the link for a no focusable element like a span". 
So I think that you only need the `<nav aria-label="pagination">` and keep the rest of the code as it was before.